spec/lib/jysonArray/jysonArray.spec.js

```diff
@@ -0,0 +1,18 @@
+const chai = require('chai');
+
+const expect = chai.expect;
+
+const jyson = require('../../../lib/jyson');
+
+describe('jysonArray.spec', () => {
+  describe('errors', () => {
+    it('must error out if not path is provided', () => {
+      try {
+        new jyson.Array();
+        return Promise.reject('an error should have been thrown');
+      } catch(error) {
+        expect(error.message).to.equal('JsonArray requires a value');
```

Comment:
always find it useful to export the error message from the actual code and then reuse in the assertion on your test. if you change the error message tomorrow your test wont fail!

Reply:
We want the tests to fail if the error message changes, that can serve as a reminder to bump a major version no?

Reply:
but the error message change and the test would be on the same version no?